packages/utils/src/addExportDateAndOriginInExcelExportData.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2020 Beyond Essential Systems Pty Ltd
+ */
+
+import moment from 'moment';
+
+/**
+ * @param {Array} exportData The sheet data that are ready to be exported
+ *
+ * @returns {Array} exportData with exported data and origin
+ */
+export const addExportedDateAndOriginAtTheSheetBottom = exportData => {
+  // Add export date and origin
+  // Add two [] for spacing between the table and the export date
+  exportData.push([], [], [`Data exported from Tupaia.org on ${moment().format('Do MMM YYYY')}`]);
```

Comment:
Usually I think it's a bit strange to modify the function parameter... could we avoid that here? One option for that:
```suggestion
  return [
    ...exportData,
    [],
    [],
    [`Data exported from Tupaia.org on ${moment().format('Do MMM YYYY')}`]
   ];
```

Reply:
Good idea!